src/main.cpp

```diff
@@ -193,8 +197,29 @@ void dump_font_info(PDFDoc *doc) {
   }
 }
 
+void pack_stream_content(Stream *stream) {
+  std::string contents;
+  int c;
+
+  stream->reset();
+  while ((c = stream->getChar()) != EOF)
+    contents += (char)c;
```

Comment:
Have you considered using [Stream::fillGooString](https://github.com/Distrotech/poppler/blob/cdb9cb2b64d70d73b2263c5aa65fafbf0ae4ec25/poppler/Stream.h#L134-L142) or an approach using `doGetChars` analogous to `fillGooString`? It may be more efficient than a per-byte function call.

Reply:
Done.